pom.xml

```diff
@@ -36,9 +36,9 @@
 	<properties>
 		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
 		<openmrsPlatformVersion>${project.version}</openmrsPlatformVersion>
-		<webservices.restVersion>2.26.0</webservices.restVersion>
+		<webservices.restVersion>2.28.0-SNAPSHOT</webservices.restVersion>
```

Comment:
we do not release with snapshot versions

Reply:
@dkayiwa i have fixed it